dev/com.ibm.ws.kernel.boot/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops

```diff
@@ -788,6 +788,14 @@ warning.timeoutWaitingForServerProcessToStop=CWWKE0968W: A timeout occurred whil
 warning.timeoutWaitingForServerProcessToStop.explanation=The server did not stop within the time allowed by the timeout value. This might be due to a slow server shutdown or a server hang.   
 warning.timeoutWaitingForServerProcessToStop.useraction=If the server is still running, consider increasing the timeout value for the 'server stop' command. On UNIX-like systems, the server stop command waits for the server process to stop. On Windows, the command waits to obtain a lock on the console.log file. Use file monitoring tools to check whether another program, such as an editor, locked the console.log file.
 
+warning.manual.PPV.env.var=CWWKE0969W: A manual PREFERRED_PLATFORM_VERSION environment variable was specified during server packaging.
+warning.manual.PPV.env.var.explanation=A manual PREFERRED_PLATFORM_VERSION environment variable was set and will not be retained during server packaging.
```

Comment:
Message guidelines state to use present tense instead of future tense, but the "will ... be" seems better in this explanation. Possibly simplify "will not be retained" to "will be deleted".